xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -834,7 +834,8 @@
       "thumbnail": { "type": "string" },
       "lastplayed": { "type": "string" },
       "broadcastnow": { "$ref": "PVR.Details.Broadcast" },
-      "broadcastnext": { "$ref": "PVR.Details.Broadcast" }
+      "broadcastnext": { "$ref": "PVR.Details.Broadcast" },
+      "uniqueid": { "type": "integer" }
```

Comment:
I know, I think Library.Id is another way of saying, this is a positive integer and required says, it should always have data.